cachito/web/models.py

```diff
@@ -231,6 +232,26 @@ def to_purl(self):
                     f"Unknown protocol in npm package version: {self.version}"
                 )
 
+        elif self.type == "pip":
+            # As per the purl spec, PyPI names should be normalized by lowercasing and
+            # converting '_' to '-'. The safe_name() function does the latter but not the
+            # former. It is not necessary to escape characters in the name, safe_name()
+            # also replaces everything except alphanumeric chars and '.' with '-'.
+            name = pkg_resources.safe_name(self.name.lower())
+            version = self.version
+
+            if version.startswith("vcs:"):
```

Comment:
Could you please give an example here for a sample `version` (just out of curiosity)?